.github/workflows/coin-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: coin-integration-alpha
+on:
+  push:
+    branches:
+      - 'int-*'
+jobs:
+  prebuild-for-platform:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, macos-latest, windows-2016]
```

Comment:
just a question, why not using `ubuntu-latest` and `windows-latest` ?

Reply:
I suggest to always use well-defined runners instead of `xxx-latest` to avoid to break CI in future latest release platforms